deploy/src/main/resources/production-marathon-prerequisites/marathon.env

```diff
@@ -0,0 +1,9 @@
+# The following properties MUST be configured before the Job Service Prerequisites can be deployed.
+
+export POSTGRES_JOB_SERVICE_DB_SERVICE_PORT=NOT_SET_POSTGRES_JOB_SERVICE_DB_SERVICE_PORT
+
+export POSTGRES_JOB_SERVICE_DB_USER=NOT_SET_POSTGRES_JOB_SERVICE_DB_USER
+export POSTGRES_JOB_SERVICE_DB_PASSWORD=NOT_SET_POSTGRES_JOB_SERVICE_DB_PASSWORD
```

Comment:
These should be consistent as @mulhollandc said, but they should consistently start with `JOB_SERVICE_` rather than `POSTGRES_` - we don't want to confuse anything that we act on with environment variables that may be applied by PostgreSQL itself.

Reply:
Made environment variables consistent throughout